constants.js

```diff
@@ -143,6 +144,7 @@ module.exports.issuesLabelCheck = issuesLabelCheck;
 module.exports.issuesAssignedCheck = issuesAssignedCheck;
 module.exports.datastoreLabelCheck = datastoreLabelCheck;
 module.exports.prLabelCheck = prLabelCheck;
+module.exports.prMilestoneCheck = prMilestoneCheck;
```

Comment:
You should probably add the `milestoned` event and specify that the `prMilestoneCheck` be run. 
See how other events like the `openEvent` were done. https://github.com/oppia/oppiabot/blob/ec404f432cf9553fbc184cc51bd572bf7e6fc806/constants.js#L71-L74

Reply:
Done